tests/resources/test_poverty_dump_input_county.json

```diff
@@ -0,0 +1,103 @@
+[["NAME","S1701_C01_001E","S1701_C02_001E","S1701_C02_002E","state","county"],
```

Comment:
I would take out "dump" from the file name of these four test data files. The "dump" versus "merge" is just for the output format and it looks like you're reusing the same input for both the merge and dump tests so there's no need to differentiate.

Reply:
Will do!